File-Formats/PDF/Working-with-DigitalSignature.md

```diff
@@ -225,39 +197,32 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Adding a digital signature using stream
 
-The following code example illustrates how to add a digital signature in the PDF document
-using stream as follows.
+The following code example illustrates how to add a digital signature in the PDF document using stream and the [PdfSignature](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfCertificate.html) class as follows.
```

Comment:
The following code example illustrates how to add a digital signature in the PDF document as stream using the [PdfSignature](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfCertificate.html) class as follows

Reply:
changed the content